.devcontainer/devcontainer.json

```diff
@@ -1,17 +1,21 @@
 // For format details, see https://aka.ms/devcontainer.json. For config options, see the
 // README at: https://github.com/devcontainers/templates/tree/main/src/postgres
 {
-	"name": "db-pluto-zola (Python 3, PostgreSQL)",
+	"name": "db-template-repo (Python 3, PostgreSQL)",
 	"dockerComposeFile": "docker-compose.yml",
 	"service": "runner",
 	"workspaceFolder": "/workspaces/${localWorkspaceFolderBasename}",
 	"customizations": {
 		"vscode": {
 			"extensions": [
-				"redhat.vscode-yaml",
 				"bierner.markdown-preview-github-styles",
+				"bierner.markdown-mermaid",
```

Comment:
I like all the new additional extensions - helps to have these set so that the vast majority of the files we create will have some sort of standard formatting